package-lock.json

```diff
@@ -121,7 +121,7 @@
     "anymatch": {
       "version": "1.3.2",
       "resolved": "https://registry.npmjs.org/anymatch/-/anymatch-1.3.2.tgz",
-      "integrity": "sha512-0XNayC8lTHQ2OI8aljNCN3sSx6hsr/1+rlcDAotXJR7C1oZZHCNsfpbKwMjRA3Uqb5tF1Rae2oloTr4xpq+WjA==",
+      "integrity": "sha1-VT3Lj5HjyImEXf26NMd3IbkLnXo=",
```

Comment:
@Shrugs is this an artifact of different npm versions? Are you using the latest npm?

Reply:
This was produced with `5.5.1` but I just updated to `5.6.0` and the output is the same. if you check out this repo, do you get a diff after `npm i`?

Reply:
Try `npm cache clean --force`, revert this file, and `npm i`.

Also note that Windows always uses `sha1` 😞  and MacOS seems to be doing this as well.

Another solution is reverting to `npm 3` as per this thread https://github.com/npm/npm/issues/17749